### DISCUSSION TITLE
iFlight Commando 8 - Lost Option for Serial Port on USB

### DISCUSSION THREAD

**Discussion by TonyNJFPV** [ExpressLRS/ExpressLRS#2428] [Q&A]
I flashed firmware from 2.8.0 to 2.9.0. From there I lost the option to select Serial Port when I plug in a USB cable.

From there I flashed back and forth a few times and now have ended up on 2.9.1 - still no serial port option. Only storage/game controller.

Not sure what to do because since I'm running ELRS 2.4.0 I need to update that over UART or ETX passthrough as WiFi gives errors for V2.5.0 V2.5.1 and V2.5.2 - so I'm sort of stuck here being unable to [upgrade up to V3.x.x of ELRS](https://www.expresslrs.org/quick-start/transmitters/iflight-commando/#). 

Any advice would be appreciated, thank you.

**Comment by lesharodin** [comment]
iFlight have External ELRS module with its own USB plug. So its not internal module, and doesnt requiere serial option in edgetx, and cant be updated via EdgeTX Passthrogh. 
![image](https://github.com/ExpressLRS/ExpressLRS/assets/12091173/3ae5c633-3d4c-42b1-86bb-f587d8e5b7af)

**Reply by TonyNJFPV** [reply]
Thanks, yes - this is how I fixed it. Appreciate the reply. In case anyone else has this question -- above is the answer! Just plug directly into the module to update.
